Gemfile

```diff
@@ -4,5 +4,4 @@ gemspec
 
 group :darwin do
   gem 'rb-fsevent', "~> 0.9.1"
-  gem 'growl_notify', "~> 0.0.3"
```

Comment:
This gem is outdated, does not seem to be used, and it is for development only. One of its dependencies refuses to build.